### PR TITLE
refactor: Simple refactoring for clean code

### PR DESCRIPTION
**Pull request by YeonwooSung**
- refactor codes for clearness

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def should_open_pr(args, info: Dict[str, Any], *, token: str="") -> bool:
             return False
         else:
             logger.warning(
-                f"Proceeding with PR creation even though there are already commits "
+                f"Proceeding with PR creation even though there are already commits "  # noqa: F541
                 "({commit_url_strs}) associated with the issue. Please only do this for your own repositories "
```
Comment (klieret) [inline run.py:187]: The real issue is that the next line is missing the `f` for the f-str

Comment (klieret) [reply run.py:187]: and if flake complains about superfluous fs, it's better to remove them than to ignore the warnings

**Review by klieret — CHANGES_REQUESTED** [review]

File: run.py (hunk 1 above)
Comment (YeonwooSung) [reply run.py:187]: Thanks for the comment. Will change it right now

File: sweagent/agent/models.py [hunk 2]
```diff
@@ -690,16 +690,21 @@ def get_model(args: ModelArguments, commands: Optional[list[Command]] = None):
...
         commands = []
 
-    if args.model_name == "human":
-        return HumanModel(args, commands)
-    if args.model_name == "human_thought":
-        return HumanThoughtModel(args, commands)
-    if args.model_name == "replay":
-        return ReplayModel(args, commands)
-    elif args.model_name.startswith("gpt") or args.model_name.startswith("ft:gpt"):
+    models = {
```
Comment (klieret) [inline sweagent/agent/models.py:693]: honestly perhaps let's just bring that back to the original version (conflicting anyway). Shouldn't make too many changes while we don't have proper CI in place

**Comment by klieret** [comment]
I think we can go ahead with this after you resolve the conflicts

**Comment by klieret** [comment]
For the future it would be better to split up fixes and refactoring (and I don't think we have enough bandwidth to welcome much refactoring, especially because we don't have much CI to test it)

**Review by klieret — APPROVED** [review]

File: sweagent/agent/models.py [hunk 3]
```diff
@@ -696,10 +696,8 @@ def get_model(args: ModelArguments, commands: Optional[list[Command]] = None):
         return HumanThoughtModel(args, commands)
     if args.model_name == "replay":
         return ReplayModel(args, commands)
-    elif args.model_name.startswith("gpt") or args.model_name.startswith("ft:gpt"):
+    elif args.model_name.startswith("gpt") or args.model_name.startswith("ft:gpt") or args.model_name.startswith("azure:gpt"):
         return OpenAIModel(args, commands)
-    elif args.model_name.startswith("claude"):
```
Comment (klieret) [inline sweagent/agent/models.py:701]: Why did you delete this?

**Review by klieret — CHANGES_REQUESTED** [review]

File: sweagent/agent/models.py (hunk 3 above)
Comment (YeonwooSung) [reply sweagent/agent/models.py]: I tried to undo the changes for the model list, and seems like I copy and pasted the code from wrong commit.. Sorry, my bad

**Review by klieret — APPROVED** [review]
